site/source/docs/porting/exceptions.rst

```diff
@@ -32,19 +23,54 @@ For example, build the following program with ``emcc -fexceptions``:
       return 0;
     }
 
-Running it will print ``throw...`` and then ``catch!``.
+The first ``throw`` will abort the program and you'll see something like this in the output:
+
+.. code-block:: text
+
+  throw...  
+  exception thrown: 5246024 - Exception catching is disabled, this exception cannot be caught. Compile with -s NO_DISABLE_EXCEPTION_CATCHING or -s EXCEPTION_CATCHING_ALLOWED=[..] to catch.
+
+If you want to opt-in, you have two following options.
+
+JavaScript-based exception support
+##################################
+
+First, you can enable exceptions via Emscripten's JavaScript-based support.
+
+To enable it, pass ``-fexceptions`` at both compile time and link time.
```

Comment:
I guess you have tested this but I just wanted to be sure, that `-fexceptions` does have the desired effect (and that we don't also need `DISABLE_EXCEPTION_CATCHING=0`).
It used to be that `-fexceptions` was the default (because with `-fno-exceptions` the frontend will refuse to compile any code with `throw` or `catch`). But since clang added the `-fignore-exceptions` we started to use that by default. So ideally we wouldn't need any `-s` flags (other than optionally `EXCEPTION_CATCHING_ALLOWED`).

Reply:
Yes, `-fexceptions` enables exception catching on Emscripten side AFAIK (and I did test it).

Reply:
It's also what the previous version of the doc said, so I don't think anything changed here except for wording.

Reply:
Thanks for confirming.